optuna/pruners/hyperband.py

```diff
@@ -24,6 +24,9 @@ class HyperbandPruner(BasePruner):
     Note that this implementation does not take as inputs the maximum amount of resource to
     a single SHA noted as :math:`R` in the paper.
 
+    This implementation can work with :class:`~optuna.samplers.TPESampler` and this combination
```

Comment:
How about modifying a bit as follows (for users that don't know what `TPESampler` is)?
```suggestion
    This implementation can work with :class:`~optuna.samplers.TPESampler`, the default sampler, and this combination
```